prefect_azure/credentials.py

```diff
@@ -382,3 +382,24 @@ def example_get_workspace_flow():
         )
 
         return workspace
+
+
+class ContainerInstanceCredentials(Block):
+    """
+    Block used to manage Azure Container Instances authentication. Stores Azure Service
+    Principal authentication data.
+    """
+
+    _block_type_slug = "container-instance-credentials"
```

Comment:
I think this can be removed so that the block slug becomes `azure-container-instance-credentials`. It's verbose, but explicit.
```suggestion
```